src/executors/priority_executor.rs

```diff
@@ -200,6 +205,173 @@ impl PriorityExecutor {
             }
         }
     }
+    
+    async fn send_bundle(
+        &self,
+        wallet: &EthereumWallet,
+        tx_request: WithOtherFields<TransactionRequest>,
+        order_hash: &str,
+        chain_id: u64,
+        target_block: u64,
+    ) -> Result<TransactionOutcome> {
+        let tx_request_for_revert = tx_request.clone();
+        
+        // Sign the transaction
+        let tx_envelope = tx_request.build(wallet).await?;
+        let raw_tx = tx_envelope.encoded_2718();
+        let signed_tx = format!("0x{}", hex::encode(&raw_tx));
+        
+        // Build bundle params (single transaction bundle)
+        let params = json!({
+            "txs": vec![signed_tx],
+            "minBlockNumber": format!("0x{:x}", target_block),
+            "maxBlockNumber": format!("0x{:x}", target_block),
```

Comment:
Just in case the tx isn't received in time, we can set the `maxBlockNumber` to `target_block` + some buffer (e.g. 4 blocks).